src/supertux/menu/main_menu.cpp

```diff
@@ -89,7 +90,7 @@ MainMenu::menu_action(MenuItem* item)
       break;
 
     case MNID_QUITMAINMENU:
-      if (true)
+      if (!g_config->confirmation_dialog)
```

Comment:
Drop the if clause and remove the previously unreachable code. I think this nags users, quit is not a dangerous action where progress is lost.